pkg/http/api.go

```diff
@@ -100,6 +101,7 @@ func NewAPI(appID string, appChannel channel.AppChannel, directMessaging messagi
 	api.endpoints = append(api.endpoints, api.constructMetadataEndpoints()...)
 	api.endpoints = append(api.endpoints, api.constructBindingsEndpoints()...)
 	api.endpoints = append(api.endpoints, api.constructHealthzEndpoints()...)
+	api.endpoints = append(api.endpoints, api.constructTransactionEndpoints()...)
```

Comment:
Can we add endpoint inside `constructStateEndpoints` function ? transaction is the part of state endpoints.